halley-core/src/main/kotlin/com/infinum/halley/core/loader/HalRelationshipLoader.kt

```diff
@@ -28,8 +28,9 @@ internal class HalRelationshipLoader : RelationshipLoader {
             try {
                 call.enqueue(callback)
                 countDownLatch.await()
-            } catch (ex: InterruptedException) {
+            } catch (interruptedException: InterruptedException) {
                 call.cancel()
+                throw interruptedException
```

Comment:
We are rethrowing the exception because otherwise the cancellation doesn’t propagate to the other calls.